test/s1/appframework/appframework_test.go

```diff
@@ -287,4 +328,70 @@ var _ = Describe("s1appfw test", func() {
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), standalonePod, esApp, false, "enabled", false, false)
 		})
 	})
+
+	Context("appframework Standalone deployment (S1) with App Framework", func() {
+		It("smoke, s1, appframework: can deploy a standalone instance with App Framework enabled and install around 350MB of apps at once", func() {
```

Comment:
Nit: This is redundant as ES app is already 2GB +?

Reply:
ES app is a special case, and installed alone. Here we install several apps together, more than just the basic apps